src/light/PointLight.ts

```diff
@@ -15,46 +15,54 @@ import type { TransformControls } from 'three/examples/jsm/Addons.js';
  * @module
  */
 
-export default class DIVEPointLight extends PointLight implements DIVESelectable, DIVEMoveable {
+export default class DIVEPointLight extends Object3D implements DIVESelectable, DIVEMoveable {
     public isMoveable: true = true;
     public isSelectable: true = true;
     public gizmo: TransformControls | null = null;
 
+    private light: PointLight;
+    private mesh: Mesh;
+
     constructor() {
```

Comment:
I would include a default true for the visibility in the constructors, what do you think?

Reply:
The default value for the 'visible'-flag comes from the parent class Object3D and is 'true' by default.